.circleci/config.yml

```diff
@@ -597,6 +600,18 @@ jobs:
       - run: *run_cmdline_tests
       - store_artifacts: *artifacts_test_results
 
+  b_win:
+    executor:
+      name: win/default # executor type
+      size: "medium" # resource class, can be "medium", "large", "xlarge", "2xlarge", defaults to "medium" if not specified
+    environment:
+      CMAKE_BUILD_TYPE: Release
+    steps:
+      - checkout
+      - run: choco install cmake -y
+      - run: mkdir build; cd build; cmake -G "Visual Studio 16 2019 Wni64" .. -DTESTS=On
+      - run: cd build; msbuild solidity.sln /p:Configuration=%CONFIGURATION% /m:%NUMBER_OF_PROCESSORS% /v:minimal
```

Comment:
It might be easier to just have cmake take care of things: ``cmake --build . --config Release``
Also if possible it might be nice to extract the whole thing into a powershell or shell script that can be used elsewhere right away...

Reply:
I just copied this over from the appveyor config for now

Reply:
It seems we need to install cmake first of all...

Reply:
Oh... if there's no cmake, then the major problem will be that there likely won't be boost either...

Reply:
Maybe there's a better base image "orb" thingy that already has both?